File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-persons_param.ts


import { ISearchPersons_Param } from './isearch-persons_param';

export class SearchPersons_Param implements ISearchPersons_Param {
  
	public SearchString: string = "";
	public CountryID?: number;
	public PageSize?: number;
	public PageNumber?: number;
	public OrderColumn: string = "";
	public OrderDirection: string = "";
	public ParticipantType: string = "";
  
}

File: GTTS/INL.SearchService/Models/SearchPersons_Param.cs

namespace INL.SearchService.Models
{
    public class SearchPersons_Param : ISearchPersons_Param
    {
        public string SearchString { get; set; }
        public int? CountryID { get; set; }
        public int? PageSize { get; set; }
        public int? PageNumber { get; set; }
        public string OrderColumn { get; set; }
        public string OrderDirection { get; set; }
        public string ParticipantType { get; set; }
    }
}
